redaxo/src/addons/cronjob/pages/cronjobs.php

```diff
@@ -159,7 +159,8 @@
     $field = $form->addSelectField('environment');
     $field->setAttribute('class', 'form-control selectpicker');
     $field->setLabel($addon->i18n('environment'));
-    $field->setNotice($addon->i18n('environment_notice'));
+    $cron_env_path = rex_path::backend().'bin/console cronjob:run';
+    $field->setNotice($addon->i18n('environment_notice',$cron_env_path));
```

Comment:
```suggestion
    $field->setNotice($addon->i18n('environment_notice', rex_path::bin('console').' cronjob:run');
```

Ich würde da keine extra variable für nutzen.
Und es gibt eine extra methode `rex_path::bin()`. (Bei uns bei yakamara liegt der bin-ordner zum beispiel gar nicht im `::backend()` ordner.)